presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -611,11 +618,18 @@ protocol::TaskInfo PrestoTask::updateInfoLocked() {
           fmt::format("drivers.{}", exec::blockingReasonToString(it.first)),
           it.second);
     }
+    if (taskStats.longestRunningOpCallMs != 0) {
+      hasStuckOperator = true;
+      addRuntimeMetricIfNotZero(
+          taskRuntimeStats,
+          "stuck_op." + taskStats.longestRunningOpCall,
+          taskStats.numCompletedDrivers);
+    }
+    // These metrics we need when we are running, so do not try to skipp them.
```

Comment:
typo `skipp`

Reply:
Thanks, since it is not a breaking typo, I'll keep it to speed up.